envs/monkey_zoo/blackbox/test_blackbox.py

```diff
@@ -186,6 +186,9 @@ def test_logout_invalidates_all_tokens(island):
 AGENT_OTP_LOGIN_ENDPOINT = "/api/agent-otp-login"
 
 
+@pytest.mark.skip(
+    reason="This test will fail if the requests cannot be made in less than 1 second."
+)
```

Comment:
This wasn't a problem before. What changed?

Reply:
It's possible that the machine type is different. I didn't record what the machine type was before redeploying it, since it should be recorded in the terraform script

Reply:
Increased resources for the windows island and the test seems to be passing now